webrender/src/frame.rs

```diff
@@ -494,6 +502,15 @@ impl Frame {
                                    iframe_scroll_layer_id,
                                    iframe_reference_frame_id);
 
+        context.builder.push_scroll_layer(iframe_reference_frame_id,
```

Comment:
> This will allow primitives to be positioned relative to the reference frame though

Please tell me that's on your list to address, since it doesn't sound like a good solution.